### PR TITLE
[css-nav-1] Update the search origin after scrolling

### PR DESCRIPTION
**Pull request by jihyerish**
The search origin will be updated only when the old focus target is completely off-screen after the scrolling.

(Note: If the old focus target is partially in the view after scrolling, the search origin won't be changed)

Related issue: https://github.com/w3c/csswg-drafts/issues/3392

### CODE REVIEW

File: css-nav-1/Overview.bs [hunk 1]
```diff
@@ -1089,17 +1089,18 @@ run the following steps:
...
     <a>actually disabled</a> or
     <a>expressly inert</a> or
     not <a>being rendered</a>,
     then let <var>searchOrigin</var> be the [=boundary box=] of <var>old focus target</var>.
 2. If <var>old focus target</var> is <a>removed</a>,
     then let <var>searchOrigin</var> be the [=boundary box=] of <var>old focus target</var> with the position of <var>old focus target</var> when it existed.
-3. Otherwise,
-    let <var>searchOrigin</var> be the viewport of <var>old focus target</var>'s <a>spatial navigation container</a>.
+3. If <var>old focus target</var> is completely off-screen,
+    then let <var>searchOrigin</var> be the viewport of <var>old focus target</var>'s <a>spatial navigation container</a>.
```
[the remark this answers is not in the document]
Comment (jihyerish) [reply css-nav-1/Overview.bs]: Yes, it's a good point! Thank you. I'll fix this.

File: css-nav-1/Overview.bs [hunk 2]
```diff
@@ -1089,17 +1089,18 @@ run the following steps:
     then return it.
 3. Otherwise, return <var>searchOrigin</var>.
 
-If the status of <var>old focus target</var> changed not by the spatial navigation,
+If the status of <var>old focus target</var> changed not by moving the focus,
 <dfn lt="update the search origin | updating the search origin">update the <a>search origin</a></dfn> as following:
-1. If <var>old focus target</var> becomes
+1. If <var>old focus target</var> is
```
[the remark this answers is not in the document]
Comment (jihyerish) [reply css-nav-1/Overview.bs]: "old focus target" may be confusing.
But when it is removed or disabled, the active element will be the Document's viewport.
So the sentence like "if activeElement is removed" seems unnatural to me.
That's the reason why I put "old" in the naming of the argument of this step.

How about "previous activeElement" or "previous focus target"?
(HTML Spec uses "focus target" instead of "active element" when it refers to the element which has the focus.)

[the remark this answers is not in the document]
Comment (hugoholgersson) [reply css-nav-1/Overview.bs]: Hmm. Ok. Yeah, I don't know. As a JS and C++ developer, when I read _activeElement_, I directly get what we're talking about. But yeah, "focus target" is perhaps more correct in a spec... Not sure. In general I think the spec is a bit too wordy :P In speech we'd just say "focus"? :)

[the remark this answers is not in the document]
Comment (jihyerish) [reply css-nav-1/Overview.bs]: I think "focus target" is more precise than just "focus". Because in this description, the object which becomes disappeared or disabled is the element, not the focus itself.  ('focus' will remain somewhere in the document)

Yeah, the spec is a bit wordy, but I think precise description is also important.

File: css-nav-1/Overview.bs (hunk 1 above)
[the remark this answers is not in the document]
Comment (jihyerish) [reply css-nav-1/Overview.bs]: I've updated the spec with [this commit.](https://github.com/w3c/csswg-drafts/commit/8ecdffb27e956e02ca2772e88e55cba1ea154e50) 
@hugoholgersson, Thank you for your opinion!
